__tests__/endpoints.test.js

```diff
@@ -21,5 +21,21 @@ describe("/api/topics", () => {
         })
     })
 })
+describe("/api", () => {
+    test(`Get 200 and a JSON object containing all the endpoint's 
```

Comment:
This test can be much simpler by requiring the endpoints file here and just comparing it to the one in the response.